source/common/runtime/runtime_features.cc

```diff
@@ -162,6 +162,9 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_google_grpc_disable_tls_13);
 // before downstream.
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_allow_multiplexed_upstream_half_close);
 
+// A flag to enable the usage of dynamic buckets for local rate limiting. For example, dynamically
+// created token buckets for each unique value of a request header.
+FALSE_RUNTIME_GUARD(envoy_reloadable_features_local_rate_limiting_with_dynamic_buckets);
```

Comment:
I think runtime flag is unnecessary because dynamic descriptors is only enabled when the empty descriptor `value` is set which is not allowed in the past.

I we can treat this is a new feature that controlled by explict API configuration.